plugins/utils/ollama.nix

```diff
@@ -0,0 +1,226 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.ollama;
+
+  actionOptionType = with helpers.nixvimTypes;
+    oneOf [
+      rawLua
+      (enum [
+        "display"
+        "replace"
+        "insert"
+        "display_replace"
+        "display_insert"
+        "display_prompt"
+      ])
+      (submodule {
```

Comment:
The submodule options won't be rendered in the documentation I think? I'm not sure if there is a way to fix that :/

Reply:
No idea...